cesium/db.go

```diff
@@ -28,12 +29,12 @@ type (
 	Frame      = core.Frame
 )
 
-var ChannelNotFound = core.ChannelNotFound
```

Comment:
Did you redefine this somewhere else. If not, we need this to be public so we can access in layers above. Note that renaming this is a breaking change.

Reply:
Re-added